calico-enterprise_versioned_docs/version-3.17/variables.js

```diff
@@ -5,7 +5,11 @@ const variables = {
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.17',
+<<<<<<< HEAD
   baseUrl: '/calico-enterprise/latest',
+=======
+  baseUrl: '/calico-enterprise/3.17',
+>>>>>>> ca381156 (Update docs for GA.)
```

Comment:
Conflict here

Reply:
I'm creating new PR to ensure I have accurate base.